second-hand-b/src/main/java/com/codesquad/secondhand/application/service/in/MemberService.java

```diff
@@ -101,4 +110,30 @@ public Member getByEmail(String email) {
         return memberRepository.findByEmail(email)
                 .orElseThrow(MemberNotFoundException::new);
     }
+
+    public Member signUpMember(String email, SignUpRequest signUpRequest) {
+        Member member = new Member(
+                email,
+                signUpRequest.getNickname(),
+                signUpRequest.getProfileImg(),
+                Role.MEMBER
+        );
+        addRegions(signUpRequest.getRegionsId(), member);
+        return save(member);
+    }
+
+    private void addRegions(List<Long> regionsId, Member member) {
+        Region firstRegion = regionService.getById(regionsId.get(REGIONS_FIRST_INDEX));
```

Comment:
Utils 로 분리하면 더 유연할 것 같아요.
```suggestion
        Region firstRegion = regionService.getById(RegionUtils.getFirstRegeionId(regionsId));
```